docs/standard/threading/toc.md

```diff
@@ -8,17 +8,17 @@
 ### [Managed and Unmanaged Threading in Windows](managed-and-unmanaged-threading-in-windows.md)
 ### [Thread.Suspend, Garbage Collection, and Safe Points](thread-suspend-garbage-collection-and-safe-points.md)
 ### [Thread Local Storage: Thread-Relative Static Fields and Data Slots](thread-local-storage-thread-relative-static-fields-and-data-slots.md)
-### [Cancellation in Managed Threads](cancellation-in-managed-threads.md)
-#### [How to: Listen for Cancellation Requests by Polling](how-to-listen-for-cancellation-requests-by-polling.md)
-#### [How to: Register Callbacks for Cancellation Requests](how-to-register-callbacks-for-cancellation-requests.md)
-#### [How to: Listen for Cancellation Requests That Have Wait Handles](how-to-listen-for-cancellation-requests-that-have-wait-handles.md)
-#### [How to: Listen for Multiple Cancellation Requests](how-to-listen-for-multiple-cancellation-requests.md)
 ## [Using threads and threading](using-threads-and-threading.md)
 ### [Creating threads and passing data at start time](creating-threads-and-passing-data-at-start-time.md)
 ### [Pausing and interrupting threads](pausing-and-resuming-threads.md)
 ### [Destroying threads](destroying-threads.md)
 ### [Scheduling threads](scheduling-threads.md)
 ### [Canceling threads cooperatively](canceling-threads-cooperatively.md)
```

Comment:
Cancelling threads cooperatively should really be a child, rather than a peer, of Cancelling Managed Threads.

Reply:
@rpetrusha indeed; thank you for review. I've updated the TOC.